webapp/pages/terms-and-conditions.vue

```diff
@@ -14,10 +15,16 @@
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-07-19T13:32:29Z" title="Friday, July 19th 2019, 3:32:29 pm +02:00">Jul 19, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,14 +1,72 @@
 <template>
   <div>
     <ds-space>
-      <ds-heading tag="h2">{{ $t('site.termsAc') }}</ds-heading>
+      <ds-heading tag="h2">{{ $t('site.termsAc') }} ( vom: {{ latestDate_TaC }})</ds-heading>
     </ds-space>
+    <div v-if="confirmed === true">
+      <ds-section class="hasConfirm" primary>
```


```suggestion
      <ds-section primary>
```
Unnecessary class

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-07-19T13:34:52Z" title="Friday, July 19th 2019, 3:34:52 pm +02:00">Jul 19, 2019</time>_

---


I think there is a redundancy here. What about using class bindings?

```suggestion
      <ds-section :primary="confirmed" :secondary="!confirmed">
```
This should make 90% of the template code obsolete.